src/main.c

```diff
@@ -102,7 +96,11 @@ static int		 output_description = 0;
 static int		 use_alternate_screen;
 static size_t		 query_size;
 static struct termios	 original_attributes;
-static struct choices	*choices;
+static struct {
+	size_t size;
+	size_t nmemb;
```

Comment:
What do you think about naming this length to be more consistent with https://github.com/thoughtbot/pick/pull/82/files#diff-803c5170888b8642f2a97e5e9423d399R106?